src/main/java/com/mongodb/jdbc/MongoJsonSchemaResult.java

```diff
@@ -0,0 +1,6 @@
+package com.mongodb.jdbc;
+
+public class MongoJsonSchemaResult {
```

Comment:
This is meant to be the result of `sqlGetResultSchema`, right? If so, the `schema` field return there is actually going to be formatted the same way as the `schema` field from `sqlGetSchema`: https://docs.mongodb.com/datalake/reference/cli/sql/sqlgetschema/#output
```
{
    ...,
    "schema": {
        "version": 1,
        "jsonSchema": ...
    }
}
```

So we need to account for that one level of nesting

Reply:
Come to think of it, I also need to deserialize the result of `sqlGetSchema` which has the same fields, `ok` and `schema`, but also one additional field, `metadata`. Will I be able to use this class for that as well?

Reply:
looks like an object with one field, `description`, which is a string. We probably don't need to actually parse it all the way though. Leaving `metadata` as `BsonDocument` should be fine.